README.md

```diff
@@ -1,2 +1,236 @@
+  with "RHV"
+
+Role Variables
+--------------
+
+### Required
+
+The following variables are required for this role to function, and are
+described in more detail below:
+
+- `he_appliance_password`
+- `he_admin_password`
+- `he_fqdn`
+- `he_vm_ip_addr`
+- `he_vm_ip_prefix`
+- `he_domain_type`
+- `he_storage_domain_addr`
+- `he_root_ssh_pubkey`
+
+
+### Passwords
+
+The RHVM instance root password, as well as the admin password for accessing
+the RHV cluster management dashboard, are set using the answerfile.
+
+This is potentially insecure, as the answerfile is sent to the RHV host
+during the hosted-engine installation, and these password will be present
+in the answerfile for the duration of the installation.
+
+- `he_appliance_password`: root password for the RHVM instance
+- `he_admin_password`: password for the "admin@internal" RHVM user
+
+### Packages
+
+- `rhv_he_deploy_rpms`: RPMs required for the hosted-engine install,
+  defaults to `['ovirt-hosted-engine-setup']`
+
+### Networking
+
+The `hosted-engine` deployment requires a preallocated IP address in the same
+subnet as the network that the RHV host will be using for RHV. Additionally,
+a fully-qualified domain name (FQDN) for that IP address must be set up in
+DNS to resolve to that address (forward and reverse). This is a pre-requisite
+to running the playbook, and cannot be done by the playbook.
+
+- `he_fqdn`: The FQDN of the RHVM instance created by hosted engine. This
+  cannot be e.g. `localhost`, `localhost.localdomain`, etc. It must be a
+  "real" and routable FQDN.
+- `he_vm_ip_addr`: The IP address given to the RHVM instance
+- `he_vm_ip_prefix`: The CIDR netmask of the subnet in which `he_vm_ip_addr`
+  resides. Again, this must match the RHV host subnet.
+- `he_bridge_if`: Optional, used to specify the interface that should be
+  added to the oVirt management bridge when multiple interfaces are configured
+  and hosted-engine cannot easily guess which interface to use.
+- `he_mgmt_network`: Optional, name of bridge to configure, to which
+  `he_bridge_if` will be attached. Defaults to `ovirtmgmt`.
+- `he_vm_mac_addr`: Optional, used if a specific MAC address is required for
+  the NIC created for the RHVM instance. Will be randomly generated if unset.
+- `he_gateway`: Optional, will be the gateway given to the RHVM instance.
+  hosted-engine will use the RHV host gateway by default.
+- `he_dns_addr`: Optional comma-delimited list of nameserver that the RHVM
+  instance will be configured to use. Will use the RHV host nameservers by
+  default.
+- `rhv_he_deploy_firewall_manager`: Optional firewall manage to use, can be
+  one of `firewalld`, `iptables`, or null (null for no firewall management).
+  Defaults to `firewalld`.
+
+### RHVM instance
+
+In addition to the networking settings above, hosted-engine exposes several
+options for configuring the RHVM instance during its creation.
+
+- `he_root_ssh_pubkey`: SSH public key to install into the root user's
```

Comment:
It sounds like this file is stored on the Ansible control machine.  If so, we may want to note that here.

Reply:
It's a string, not a file, so it can come in however you want. I'll update the comment to make that clear.